TimeOfFlight/include/TOFEstimators.h

```diff
@@ -3,22 +3,36 @@
 
 #include "marlin/Processor.h"
 #include "lcio.h"
+
+//std stuff
+#include <iostream>
 #include <string>
 #include <vector>
+#include <random>
+#include <functional>
+#include <memory>
+using std::string, std::vector, std::endl;
+
 #include <gsl/gsl_rng.h>
 
+#include <EVENT/Track.h>
+#include <EVENT/Cluster.h>
+
+#include "DDRec/Vector3D.h"
+using dd4hep::rec::Vector3D;
```

Comment:
Similar thing here. Potentially even more dangerous, because `Vector3D` is more likely to be a user defined class, so "polluting" the global namespace with this might lead to name clashes.